packages/ui/src/components/Designer/Canvas/index.tsx

```diff
@@ -247,7 +253,15 @@ const Main = (props: Props, ref: Ref<HTMLDivElement>) => {
     .every((schema) => schema.rotate !== undefined);
 
   return (
-    <div ref={ref} style={{ overflow: 'overlay' }}>
```

Comment:
`overflow: overlay` was deprecated a while ago and does not have consistent support. We're using `overflow: scroll` now where needed.